source/includes/tasks/_header.md

```diff
@@ -0,0 +1 @@
+# Tasks
```

Comment:
tasks files aren't shown

Reply:
am i right in thinking this comment means although the files are there, they aren't in the API menu so can't be accessed?

Reply:
yes. as with above, i can delete them completely or leave in case we need them in future